python/py-line_profiler/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-line_profiler
+version             4.1.2
+revision            0
+supported_archs     noarch
+platforms           {darwin any}
+license             BSD
+maintainers         nomaintainer
+
+description         Line-by-line profiler
+long_description    {*}${description}
+
+homepage            https://github.com/pyutils/line_profiler/tree/main
+
+checksums           rmd160  a40f5b49a756447c21da925ddc764fcac93f7d79 \
+                    sha256  aa56578b0ff5a756fe180b3fda7bd67c27bbd478b3d0124612d8cf00e4a21df2 \
+                    size    78149
+
+python.versions     38 39 310 311 312
+
+if {${name} ne ${subport}} {
+    depends_lib-append  port:py${python.version}-cython
```

Comment:
tupically this is only a build dependency - why is it here a library dependency?

Reply:
Thank you. Fixed.